napari/_qt/_qapp_model/qactions/_view.py

```diff
@@ -0,0 +1,91 @@
+from typing import List
+
+from app_model.types import Action, KeyCode, KeyMod, StandardKeyBinding
+
+from ...._app_model.constants import CommandId, MenuGroup, MenuId
+from ....settings import get_settings
+from ....utils.translations import trans
+from ...qt_main_window import Window
+from ...qt_viewer import QtViewer
+
+
+def _tooltip_visibility_toggle():
+    settings = get_settings().appearance
+    settings.layer_tooltip_visibility = not settings.layer_tooltip_visibility
```

Comment:
Test for this should be simple. I know that it does not look like an important change, but I prefer to have a simple test rather than have not covered lines. This is also showing a good pattern for new contributors.

Reply:
can do